server/api/embedded_form.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+type EmbeddedForm struct {
```

Comment:
1. 1/5 maybe we should stop using the term `Form` here as it may be confusing with `api.Form`, and replace it with something like `type InPost` or `Embedded`
2. 0/5 maybe we should use the `AppID` in the binding - this way in theory we can attach bindings from different apps to a single Post. It's an esoteric possibility, but I'd rather not preclude it in he data model.

Reply:
1. I tried to start a discussion about naming in community: https://community.mattermost.com/core/pl/zgn3medxujfcj84igbxxzcfxge . When we get to a decission I will make the changes.
2. Right now we have no way to impede posting a form that will perform an action on a different app. I feel that is a security vector we should control. That is why I wanted to put that on a top level. Nevertheless, if we want to go rogue in that direction, I don't mind using the id on the binding.